examples/summarization/t5/README.md

```diff
@@ -0,0 +1,25 @@
+***This script evaluates the [T5 Model](https://arxiv.org/pdf/1910.10683.pdf) ``t5-large`` on the CNN/Daily Mail test dataset. Please note that the results in the paper were attained using a ``t5-large`` model fine-tuned on summarization, so that results will be slightly worse here***
```

Comment:
1. Suggest changing "the [T5 Model](https://arxiv.org/pdf/1910.10683.pdf) ``t5-large``" to "the multitask pre-trained checkpoint for [``t5-large``](https://arxiv.org/pdf/1910.10683.pdf)
2. If you want to be more specific than "slightly", it looks like they are about 0.5 ROUGE points (on all ROUGE variants) lower than they would be with fine-tuning.